aws/terraform/README.md

```diff
@@ -0,0 +1,112 @@
+## Pre-requisites
+
+### Terraform
+
+[Download](https://www.terraform.io/downloads.html) and install terraform 0.14.x.
+
+### AWS
+
+AWS should be [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) with a profile. The `AWS_PROFILE` environment should be set to the profile that has been configured.
+
+The [awscli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) can be used as follows to 
+confirm that it has been configured properly by running:
+
+```
+aws sts get-caller-identity
+```
+
+which should output your account information.
+
+## Setup
+
+### Infrastructure stack
+
+The infra sub-project provides some pre-requisite infrastructure for the metaflow service. For more details see the [README](aws/terraform/infra/README.md)
+
+Copy `example.tfvars` to `prod.tfvars` (or whatever environment name you prefer) and update that `env` name and the `region` as needed.
+
+Initialize the terraform:
+
+`cd infra && terraform init`
+
+Apply it:
+
+```
+terraform apply --var-file prod.tfvars
+```
+
+### Metaflow stack
+
+The metaflow sub-project provisions the metadata API, step functions, and an AWS Batch queue. For more details see the 
+[README](aws/terraform/metaflow/README.md)
+
+Copy `example.tfvars` to `prod.tfvars` (or whatever environment name you prefer) and update that `env` name and the `region` as needed.
```

Comment:
L40:43 is repeated above.

Reply:
I believe this on purpose but the confusion means that this isn't written clearly. The `terraform/infra` and the `terraform/metaflow` project will each have their own `.tfvars`, which in the documentation as it stands are both called `prod.tfvars`.

This is because the `infra` and `metaflow` project define their own set of variables in `variables.tf` and utilize separate `.tfvars` to allow the developer to define values. 

Does that make sense or perhaps I missed something